.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint
+
+on: push
+
+jobs:
+  pipeline:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Linter
+        run: pip install black
+        
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@stable
```

Comment:
This is totally separate image with black preinstalled - why are we using that one instead of the one installed  using pip/pipenv?